store/postgres/migrations/2020-04-10-111111_add_deployment_errors/up.sql

```diff
@@ -0,0 +1,54 @@
+-- create subgraph_error
+create table subgraphs."subgraph_error" (
+        "id"                 text not null,
+        "subgraph_id"        text not null,
```

Comment:
Could we add a `references subgraphs.subgraph_deployment on delete cascade`? That way, when we clean up subgraph deployments we don't need to delete errors manually.

Reply:
Awesome! I'll do this.

Reply:
PG tells me this would take a unique constraint on `subgraphs.subgraph_deployment(id)`. Should I add such a constraint?

Reply:
Yes, I think that's a good thing in any event: for normal entities, we can't have a unique constraint on `id`, but for these metadata entities, we actually want to make sure they are unique.

Reply:
Done!